web_timeline/readme/CREDITS.rst

```diff
@@ -0,0 +1,4 @@
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```

Comment:
You can :fire: this file. No need to give credits to OCA for a module in OCA IMHO.